_api-reference/security/configuration/update-configuration.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: default
+title: Update Security Configuration API
+parent: Configuration APIs
+grand_parent: Security APIs
+nav_order: 30
+---
+
+# Update Security Configuration API
+**Introduced 1.0**
+{: .label .label-purple }
+
+The Update Security Configuration API creates or updates the Security plugin's configuration directly through the REST API. This configuration manages core security settings including authentication methods, authorization rules, and access controls.
+
+This operation can easily break your existing security configuration. We strongly recommend using the `securityadmin.sh` script instead, which includes validations and safeguards to prevent misconfiguration.
+{: .warning}
+
+<!-- spec_insert_start
+api: security.update_configuration
+component: endpoints
+-->
+## Endpoints
+```json
+PUT /_plugins/_security/api/securityconfig/config
+```
+<!-- spec_insert_end -->
+
+## Request body fields
+
+The request body is **required**. It is a JSON object with the following fields:
+
+| Property | Required | Data type | Description |
+| :--- | :--- | :--- | :--- |
+| `dynamic` | **Required** | Object | The main configuration object containing all security configuration settings. |
+
+<details markdown="block">
+  <summary>
+    Request body fields: <code>dynamic</code>
+  </summary>
+  {: .text-delta}
+
+`dynamic` is a JSON object with the following fields:
+
+| Property | Data type | Description |
+| :--- | :--- | :--- |
+| `auth_failure_listeners` | Object | Configuration for handling authentication failures, including thresholds and actions. |
+| `authc` | Object | Authentication configuration domains, defining how users are authenticated. |
+| `authz` | Object | Authorization configuration, defining how permissions are evaluated. |
+| `disable_intertransport_auth` | Boolean | When `true`, disables authentication for internal node-to-node communication. |
```

Comment:
Let's not include this setting. If this has not yet been removed then it should be removed. Its a noop.